packages/core/src/Parser.ts

```diff
@@ -10,16 +10,16 @@ import { VElement } from './VNodes/VElement';
 import { FragmentNode } from './VNodes/FragmentNode';
 import { FormatType, CharNode } from '../../plugin-char/CharNode';
 
+export type ParsingMap = Map<VNode, Node[]>;
 export interface ParsingContext {
     readonly rootNode?: Node;
     currentNode?: Node;
-    parsingMap?: Map<VNode, Node[]>;
     parentVNode?: VNode;
     format?: FormatType[];
     vDocument: VDocument;
 }
 export type ParsingPredicate = (node: Node) => ParsingFunction;
-export type ParsingFunction = (context: ParsingContext) => ParsingContext;
+export type ParsingFunction = (context: ParsingContext) => [ParsingContext, ParsingMap];
```

Comment:
This is undoing the previous commit. In a good way so there is that, but I don't understand why it is not squashed into the previous commit.